isilon_create_users.sh

```diff
@@ -213,125 +213,123 @@ echo "# use this file to add to the group file of your clients" | cat > $grpfile
+       gid=$(getGidFromGroup $user $ZONE)
+       echo "$user:x:$uid:$gid:hadoop-svc-account:/home/$user:/bin/bash" | cat >> $passwdfile
+       [ $? -ne 0 ] && addError "Could not create entry in passwd file stub $passwdfile for $user with uid $uid"
+    fi
+    uid=$(( $uid + 1 ))
 done
 # set +x
 
 for group in $SUPER_GROUPS; do
-    group="$group$CLUSTER_NAME"
-    sprgrp=`grep $group $grpfile`
-    [ $? -ne 0 ] && addError "Could not locate entry $group in group file stub $grpfile"
-    for user in $SUPER_USERS; do
-        user="$user$CLUSTER_NAME"
-        isi auth groups modify $group --add-user $user --zone $ZONE
-        [ $? -ne 0 ] && addError "Could not add user $user to $group group in zone $ZONE"
-        sprgrp="$sprgrp,$user"
-    done
-    sed -i .bak /$group/d $grpfile
-    echo $sprgrp | cat >> $grpfile
+    group="$group$CLUSTER_NAME"
+    sprgrp=`grep $group $grpfile`
+    [ $? -ne 0 ] && addError "Could not locate entry $group in group file stub $grpfile"
+    for user in $SUPER_USERS; do
+        user="$user$CLUSTER_NAME"
+        isi auth groups modify $group --add-user $user --zone $ZONE
+        [ $? -ne 0 ] && addError "Could not add user $user to $group group in zone $ZONE"
+        sprgrp="$sprgrp,$user"
+    done
+    sed -i .bak /$group/d $grpfile
+    echo $sprgrp | cat >> $grpfile
 done
 # set +x
 
 for proxy in $PROXY_SUPER; do
-    proxy="$proxy$CLUSTER_NAME"
-    isi hdfs proxyusers create --proxyuser $proxy --add-group hadoop$CLUSTER_NAME --add-user $SMOKE_USER$CLUSTER_NAME --zone $ZONE
-    [ $? -ne 0 ] && addError "Could not create proxyuser $proxy in zone $ZONE"
+    proxy="$proxy$CLUSTER_NAME"
+    isi hdfs proxyusers create --proxyuser $proxy --add-group hadoop$CLUSTER_NAME --add-user $SMOKE_USER$CLUSTER_NAME --zone $ZONE
+    [ $? -ne 0 ] && addError "Could not create proxyuser $proxy in zone $ZONE"
 done
 
 for proxy in $PROXY_USERONLY; do
-    proxy="$proxy$CLUSTER_NAME"
-    isi hdfs proxyusers create --proxyuser $proxy --add-user $SMOKE_USER$CLUSTER_NAME --zone $ZONE
-    [ $? -ne 0 ] && addError "Could not create proxyuser $proxy in zone $ZONE"
+    proxy="$proxy$CLUSTER_NAME"
+    isi hdfs proxyusers create --proxyuser $proxy --add-user $SMOKE_USER$CLUSTER_NAME --zone $ZONE
+    [ $? -ne 0 ] && addError "Could not create proxyuser $proxy in zone $ZONE"
 done
 
 # Special cases
 case "$DIST" in
-    "cdh")
-        #Deal with special cases on Isilon 
-	isi auth groups modify sqoop$CLUSTER_NAME --add-user sqoop2$CLUSTER_NAME --zone $ZONE
-        [ $? -ne 0 ] && addError "Could not add user sqoop2$CLUSTER_NAME to sqoop$CLUSTER_NAME group in zone $ZONE"
-        isi auth groups modify hive$CLUSTER_NAME --add-user impala$CLUSTER_NAME --zone $ZONE
-        [ $? -ne 0 ] && addError "Could not add user implala$CLUSTER_NAME to hive$CLUSTER_NAME group in zone $ZONE"
-	isi auth groups modify hadoop$CLUSTER_NAME --add-user hbase$CLUSTER_NAME --zone $ZONE
```

Comment:
Is this not needed for users who skip `isilon_create_directories.sh` on 8.0.1.x anymore? https://github.com/Isilon/isilon_hadoop_tools/commit/80b8cd7da542f9e5550b6ccfd6d61bd4ef822d7b